x/epoching/keeper/staking_functions.go

```diff
@@ -5,6 +5,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/babylonchain/babylon/x/epoching/types"
 )
 
 // CheckMsgCreateValidator performs stateless checks on a given `MsgCreateValidator` message
```

Comment:
```suggestion
// CheckMsgCreateValidator performs checks on a given `MsgCreateValidator` message
```

Now this function does not only perform stateless checks anymore